### DISCUSSION TITLE
How to delete it

### DISCUSSION THREAD

**Discussion by itomega6** [ohmyzsh/ohmyzsh#12567] [Help]
do you know how to delete (( this with root@parrot)) ?
![IMG_٢٠٢٤٠٧٢١_٢٢٢٥١١](https://github.com/user-attachments/assets/c4b7cfff-56a0-4dfb-bb4c-da5434adb0af)

**Comment by felipesbonatti** [comment]
Can you detail it?

**Comment by roeniss** [comment]
I think `RPROMT` (or right prompt) is what you're looking for.
